hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -354,18 +355,26 @@ class DashboardSystemStatus(t.TypedDict):
 
 
 def get_dashboard_hashes() -> DashboardCount:
-    return DashboardCount(
-        total=34217123456,
-        today=145609278,
-        updated_at="MockData and Timestamp",
-    )
+    return get_count_from_record_cls(PipelinePDQHashRecord)
 
 
 def get_dashboard_matches() -> DashboardCount:
+    return get_count_from_record_cls(PDQMatchRecord)
+
+
+def get_count_from_record_cls(
+    record_cls: t.Union[t.Type[PDQMatchRecord], t.Type[PipelinePDQHashRecord]]
+) -> DashboardCount:
+    # TODO this is an very inefficient way to do this but connecting to real counts > mock
```

Comment:
Any estimate of how long the page currently takes to load?

Reply:
I don't think it is a bottleneck right now this is more a warning for when we get a lot of hashes/matches in a system it might become one.

Reply:
(the mocked data Action card was the last to populate on page load when I tested it just now)